frontend/src/views/ManagementPage/CanteenCard.vue

```diff
@@ -78,6 +84,12 @@ export default {
         }
       }
     },
+    usesCentralKitchenDiagnostics() {
+      return (
+        this.canteen.productionType === "site_cooked_elsewhere" &&
+        this.canteen.centralKitchenDiagnostics?.find((d) => d.year === this.teledeclarationYear)
```

Comment:
Isn't it possible still for a CC to declare just the appro, and for the satellite to declare the rest ?

Reply:
Yes it is, this is a first-step PR. We're in discussions with André on how to display these cases (https://github.com/betagouv/ma-cantine/issues/2578#issuecomment-1527515269)